util/cov.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/bash
+
+# This run `kcov` on Clippy. The coverage report will be at
+# `./target/cov/index.html`.
+# `compile-test` is special. `kcov` does not work directly on it so these files
+# are compiled manually.
+
+tests=(
+    camel_case
+    cc_seme
+    consts
+    dogfood
+    issue_825
+    matches
+    trim_multiline
+    used_underscore_binding_macro
+    versioncheck
+)
+tmpdir=$(mktemp -d)
+
+cargo test --no-run --verbose
+
+for t in "${tests[@]}"; do
+  kcov \
+    --verify \
+    --include-path="$(pwd)/src","$(pwd)/clippy_lints/src" \
+    "$tmpdir/$t" \
+    cargo test --test "$t"
```

Comment:
wonder if there's a better way of doing this. perhaps we could turn off the compiletest tests with an env var, and then just run `cargo test`?